packages/tools/src/types/EventTypes.ts

```diff
@@ -201,6 +222,28 @@ type MouseDownEventDetail = NormalizedMouseEventDetail & {
   deltaPoints: IPoints;
 };
 
+/**
+ * EventDetail for touchstart event
+ */
+type TouchStartEventDetail = NormalizedTouchEventDetail & {
```

Comment:
Please start with the MouseStartEventDetail, and then add specific behaviour that identifies this as a touch event.  That allows leaving most of the tools classes alone, and only tweaking them if needed.  There should/must be an identifier that identifies this as a touch event, and probably something that identifies what "drag" type to listen to.